### PR TITLE
docs(bandwidth-manager): add note on per-pod limits

### PR DESCRIPTION
**Pull request by raphink**
Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

### CODE REVIEW

File: Documentation/installation/bandwidth-manager.rst [hunk 1]
```diff
@@ -170,6 +170,14 @@ the ``netperf-server`` Pod):
 Each Pod is represented in Cilium as an :ref:`endpoint` which has an identity. The above
 identity can then be correlated with the ``cilium endpoint list`` command.
 
+.. note::
+
+   Bandwidth limits apply on a per-pod scope. Each pod in the identity gets its own
+   bandwidth allocation.
```
Comment (pchaigno) [inline Documentation/installation/bandwidth-manager.rst]: This is confusing as there is only one Cilium Endpoint Identity (such as 491 above) per Cilium-managed pod.

**Review by pchaigno — CHANGES_REQUESTED** [review]

File: Documentation/installation/bandwidth-manager.rst [hunk 2]
```diff
@@ -170,6 +170,14 @@ the ``netperf-server`` Pod):
 Each Pod is represented in Cilium as an :ref:`endpoint` which has an identity. The above
 identity can then be correlated with the ``cilium endpoint list`` command.
 
+.. note::
+
+   Bandwidth limits apply on a per-pod scope. Each pod in the identity gets its own
```
Comment (borkmann) [inline Documentation/installation/bandwidth-manager.rst]: s/pod/Pod/

File: Documentation/installation/bandwidth-manager.rst [hunk 3]
```diff
@@ -170,6 +170,14 @@ the ``netperf-server`` Pod):
...
+   bandwidth allocation.
+   
+   In our example, if the identity applies to two pods (with the same labels),
+   the total egress bandwidth will be 20Mbps.
```
Comment (borkmann) [inline Documentation/installation/bandwidth-manager.rst]: Imho, this is too confusing. Maybe:
```
In our example, if multiple replicas of the Pod are created, then each of the Pod instances receives a 10M bandwidth limit.
```

**Review by pchaigno — APPROVED** [review]

**Review by NikAleksandrov — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]

**Comment by qmonnet** [comment]
Backport note: the file was under `Documentation/gettingstarted/` in previous Cilium versions.

**Review by borkmann — APPROVED** [review]

**Pull request merged**